include/boost/json/detail/monotonic_resource.hpp

```diff
@@ -0,0 +1,91 @@
+//
+// Copyright (c) 2020 Krystian Stasiowski (sdkrystian@gmail.com)
+//
+// Distributed under the Boost Software License, Version 1.0. (See accompanying
+// file LICENSE_1_0.txt or copy at http://www.boost.org/LICENSE_1_0.txt)
+//
+// Official repository: https://github.com/vinniefalco/json
+//
+
+#ifndef BOOST_JSON_DETAIL_MONOTONIC_RESOURCE_HPP
+#define BOOST_JSON_DETAIL_MONOTONIC_RESOURCE_HPP
+
+#include <boost/json/detail/config.hpp>
+
+#include <cstddef>
+
+namespace boost {
+namespace json {
+namespace detail {
+namespace dummy {
```

Comment:
I don't like the word "dummy" appearing in my source code. Use `other` or something sterile.

Reply:
How about `empty_types`

Reply:
what's wrong with `other` ?

Reply:
It provides no information as to what it contains